Sources/TokamakDOM/Views/HTML.swift

```diff
@@ -42,10 +42,21 @@ extension AnyHTML {
     """
   }
 
-  func update(dom: JSObjectRef) {
-    // FIXME: handle attributes and listeners here
+  func update(dom: DOMNode) {
+    // FIXME: is there a sensible way to diff attributes and listeners to avoid
+    // crossing the JavaScript bridge and touching DOM if not needed?
+
+    // @carson-katri: For diffing, could you build a Set from the keys and values of the dictionary,
+    // then use the standard lib to get the difference?
```

Comment:
To do this I'd suggest `zip`, but tuples aren't `Hashable`, so maybe something like:
```swift
fileprivate struct KVP : Hashable {
    let key: String
    let value: String
}
...
Set(attributes.map { KVP(key: $0.0, value: $0.1) })
.subtracting(...)
```

Reply:
Totally, I just added that as a comment so that diffing is handled as a separate PR. Maybe even in the future when we have some kind of performance benchmarks so that we can carefully tweak for performance.